modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+[id="odc-creating-a-binding-connection-between-components_{context}"]
+== Creating a binding connection between components
+
+You can establish a binding connection with Operator-backed components.
+
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
+The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+
+.Prerequisites
+
+* You have created and deployed a Spring PetClinic sample application that uses the *Developer* perspective.
+* You have installed the {servicebinding-title} from the OperatorHub.
+* You have installed the *Crunchy Postgres for Kubernetes* Operator from the OperatorHub that uses `v5` *Update* channel.
+* You have created and deployed a Crunchy PostgreSQL database instance that uses the *Developer* perspective. This instance has the following four components: `hippo-backup`, `hippo-instance`, `hippo-repo-host`, and `hippo-pgbouncer`.
+
+.Procedure
+
+. Switch to the *Developer* perspective and ensure that you are in the appropriate project, for example, `my-petclinic`. In the *Topology* view, hover over the Spring PetClinic sample application to see a dangling arrow on the node.
+. Click and drag the arrow towards the *hippo* database in the Postgres Cluster to make a binding connection with the Spring PetClinic sample application. A *Create Service Binding* dialog box appears.
+.. Enter the name and click *Create*.
++
+.Service Binding modal
+image::odc-sbc-modal.png[]
+
+
+Alternatively, in the *+Add* view, click the *YAML* option to see the *Import YAML* screen. Use the YAML editor and add the `ServiceBinding` resource:
+
+[source,YAML]
+----
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-pgcluster
+    namespace: my-petclinic
+spec:
+    services:
+    - group: postgres-operator.crunchydata.com
+      version: v1beta1
+      kind: PostgresCluster
+      name: hippo
+    application:
+      name: spring-petclinic
+      group: apps
+      version: v1
+      resource: deployments
+----
+
+A service binding request is created and the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
+
+.Binding connector
+image::odc-binding-connector.png[]
```

Comment:
This is a description of the outcome rather than a step, and doesn't belong inside the procedure. It expands upon the information in line 8. Consider combining this paragraph and illustration with line  8.  
```suggestion
```

Reply:
It will be covered as part of https://issues.redhat.com/browse/RHDEVDOCS-4234